app/controllers/api/v1/session.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'barong/security/access_token'
```

Comment:
This should never be required in such way in rails app. Code it in such way it is eager loaded from lib.